screens/trainer/trainer.gui_script

```diff
@@ -112,15 +112,15 @@ local function update_profile_name(self)
 end
 
 
-local function update_checkbox(self, checkbox)
+local function update_checkbox(checkbox, name)
```

Comment:
This function name is misleading because it only works for “always stab type” checkboxes.